man/systemd.exec.xml

```diff
@@ -897,14 +897,14 @@
         in which case all paths listed will have limited access from within the namespace. If the empty string is
         assigned to this option, the specific list is reset, and all prior assignments have no effect.</para>
 
-        <para>Paths in <varname>ReadOnlyPaths=</varname> and <varname>InaccessiblePaths=</varname> may be prefixed with
-        <literal>-</literal>, in which case they will be ignored when they do not exist. Note that using this setting
-        will disconnect propagation of mounts from the service to the host (propagation in the opposite direction
-        continues to work). This means that this setting may not be used for services which shall be able to install
-        mount points in the main mount namespace. Note that the effect of these settings may be undone by privileged
-        processes. In order to set up an effective sandboxed environment for a unit it is thus recommended to combine
-        these settings with either <varname>CapabilityBoundingSet=~CAP_SYS_ADMIN</varname> or
-        <varname>SystemCallFilter=~@mount</varname>.</para></listitem>
+        <para>Paths in <varname>ReadWritePaths=</varname>, <varname>ReadOnlyPaths=</varname> and
+        <varname>InaccessiblePaths=</varname> may be prefixed with <literal>-</literal>, in which case they will be ignored
+        when they do not exist. Note that using this setting will disconnect propagation of mounts from the service to
+        the host (propagation in the opposite direction continues to work). This means that this setting may not be used
+        for services which shall be able to install mount points in the main mount namespace. Note that the effect of
+        these settings may be undone by privileged processes. In order to set up an effective sandboxed environment for
+        a unit it is thus recommended to combine these settings with either
+        <varname>CapabilityBoundingSet=~CAP_SYS_ADMIN</varname> or <varname>SystemCallFilter=~@mount</varname>.</para></listitem>
```

Comment:
`InaccessiblePaths=/run/systemd/private /run/dbus/system_bus_socket`

 ```
-bash-4.3# systemctl start hola
-bash-4.3# systemctl status hola
● hola.service
   Loaded: loaded (/etc/systemd/system/hola.service; static; vendor preset: disabled)
   Active: failed (Result: exit-code) since Sat 2016-09-24 13:39:43 CEST; 3s ago
  Process: 102 ExecStart=/usr/bin/systemd-run sh -x -c echo 0 > /proc/latency_stats; sleep 10; echo hhhhh; ls -lha /run/systemd/; (code=exited, status=1/FAILURE)
 Main PID: 102 (code=exited, status=1/FAILURE)

Sep 24 13:39:43 fedora-tree-24 systemd[1]: Started hola.service.
Sep 24 13:39:43 fedora-tree-24 systemd-run[102]: Failed to create bus connection: Connection refused
Sep 24 13:39:43 fedora-tree-24 systemd[1]: hola.service: Main process exited, code=exited, status=1/FAILURE
Sep 24 13:39:43 fedora-tree-24 systemd[1]: hola.service: Unit entered failed state.
Sep 24 13:39:43 fedora-tree-24 systemd[1]: hola.service: Failed with result 'exit-code'.
```

So you have to protect the bus APIs also the user bus... in my setup I already do this, however lot of other tools don't do it... and I already discussed this @poettering we may need later a simple option `ProtectBus...` to abstract this...
with previous kdbus, the setup was bind mount custom endpoints on top of default bus to only show some predefined bus names... all the rest will be hidden. However these are unrelated details... which should not conflict with these settings. Of course we may need to document this for now.

> Do we really need these settings?

Sure, we need them for our users and a better integration.